src/avatar/src/Avatar.js

```diff
@@ -156,7 +77,8 @@ class Avatar extends PureComponent {
         justifyContent="center"
         backgroundColor={colorProps.backgroundColor}
         title={name}
-        {...props}
+        innerRef={ref}
```

Comment:
We'll need to update this to `ref` once https://github.com/segmentio/evergreen/pull/843 lands :)